sections/futures/PositionCard/ClosePositionModal.tsx

```diff
@@ -108,7 +108,9 @@ const ClosePositionModal: FC<ClosePositionModalProps> = ({
 			},
 			{
 				label: t('futures.market.user.position.modal-close.size'),
-				value: formatCurrency(currencyKey, position?.size ?? zeroBN, { currencyKey }),
+				value: formatCurrency(currencyKey || '', position?.size ?? zeroBN, {
+					sign: currencyKey[0] === 's' ? currencyKey.slice(1) : currencyKey,
```

Comment:
It works, but you may use the method in [currencies.ts](https://github.com/Kwenta/kwenta/blob/v2-dev/utils/currencies.ts#L13-L14), so it's easier to refactor the code in the future.

Reply:
Hey @LeifuChen I updated this now. Thanks.